Sources/PostgresNIO/New/Messages/SSLRequest.swift

```diff
@@ -0,0 +1,21 @@
+import NIO
+
+extension PSQLFrontendMessage {
+    /// A message asking the PostgreSQL server if SSL is supported
```

Comment:
Replace "SSL" with "TLS" here. (But not below, as those refer to the message definition from PSQL's protocol.)